types/src/jwks/unsupported.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright © Aptos Foundation
+
+use aptos_crypto_derive::{BCSCryptoHash, CryptoHasher};
+use serde::{Deserialize, Serialize};
+use aptos_crypto::HashValue;
+use move_core_types::value::{MoveStruct, MoveValue};
+use crate::move_any::AsMoveAny;
+use crate::move_utils::as_move_value::AsMoveValue;
+
+/// Move type `0x1::jwks::UnsupportedJWK` in rust.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, CryptoHasher, BCSCryptoHash)]
+pub struct UnsupportedJWK {
```

Comment:
Why do we need this? Is this a blacklist?

Reply:
We want to also record unsupported jwks for public observability: we want to know when the providers start using a new key format.